service/controller/resource/app/desired.go

```diff
@@ -182,6 +222,16 @@ func (r *Resource) newAppSpecs(ctx context.Context, cr apiv1alpha2.Cluster) ([]k
 			}
 		}
 
+		if val, ok := userOverrideConfigs[app.App]; ok {
```

Comment:
Can you add a comment so it's clear this is for overriding the config for manual testing? 

Similar to the comment above about overriding the app config.